pyproject.toml

```diff
@@ -246,5 +246,5 @@ known-first-party = ["gradient", "tests"]
 [tool.ruff.lint.per-file-ignores]
 "bin/**.py" = ["T201", "T203"]
 "scripts/**.py" = ["T201", "T203"]
-"tests/**.py" = ["T201", "T203"]
+"tests/**.py" = ["T201", "T203", "ARG001"]
```

Comment:
This is a pretty significant thing to disable, why can't this be ignored inline?